torch/distributed/rpc/__init__.py

```diff
@@ -14,7 +14,7 @@ def init_model_parallel(self_name,
                             backend=RpcBackend.PROCESS_GROUP,
                             self_rank=-1,
                             init_method=None,
-                            num_send_recv_threads=4):
+                            num_send_recv_threads=4, rpc_timeout=100):
```

Comment:
rpc_timeout should be specified by the user in milliseconds and not seconds. Some users might want to set it to 100ms for example.

Reply:
Can we enforce that the user facing API only accepts `timedelta` as the timeout parameter? This would make the API much cleaner.

Reply:
Also probably rename it to rpc_timeout_ms